app/views/contexts/grand_calculated_summary_context.py

```diff
@@ -0,0 +1,126 @@
+from typing import Iterable, Mapping
+
+from werkzeug.datastructures import ImmutableDict
+
+from app.questionnaire.questionnaire_schema import (
+    get_calculation_block_ids_for_grand_calculated_summary,
+)
+from app.views.contexts.calculated_summary_context import CalculatedSummaryContext
+from app.views.contexts.summary.group import Group
+
+
+class GrandCalculatedSummaryContext(CalculatedSummaryContext):
+    def _build_grand_calculated_summary_section(
+        self, rendered_block: ImmutableDict
+    ) -> dict:
```

Comment:
Here we need top level for the dict.

Reply:
added for return type but rendered_block would be str | Any so left that as is